zarr/core.py

```diff
@@ -1112,7 +1147,7 @@ def __setitem__(self, selection, value):
 
         See Also
         --------
-        get_basic_selection, set_basic_selection, get_mask_selection, set_mask_selection,
+        basic_selection, set_basic_selection, get_mask_selection, set_mask_selection,
```

Comment:
Is there some reason this needed to change? If not, may make sense to revert.

```suggestion
        get_basic_selection, set_basic_selection, get_mask_selection, set_mask_selection,
```